### PR TITLE
Isolate tasks in bootstrap.bat

### PR DESCRIPTION
**Pull request by samiebee43**
Bring bootstrap debugging feature to mzeig-testing-4.0.6

### CODE REVIEW

**Review by mzieg — APPROVED** [review]
Changes makes sense.  My impression is that this will no longer have a "one-shot build installer" (equivalent to the old "installer" argument), but will rather require at least three invocations to match the old behavior (once with "rebuildall", then "pyinstaller", then "innosetup")?

File: scripts/bootstrap.bat [hunk 1]
```diff
@@ -178,45 +233,48 @@ REM echo %date% %time% ======================================================
...
-if %errorlevel% neq 0 goto script_failure
-
-echo.
-echo %date% %time% ======================================================
-echo %date% %time% Logging Conda packages
-echo %date% %time% ======================================================
-echo.
-cmd /c "conda list --explicit" 
-if %errorlevel% neq 0 goto script_failure
+if "%install_python_deps%" == 1 (
```
Comment (mzieg) [inline scripts/bootstrap.bat:236]: I think this needs to be "1"

File: scripts/bootstrap.bat [hunk 2]
```diff
@@ -226,106 +284,99 @@ echo.
...
 echo PYTHONPATH = %PYTHONPATH%
 
-echo.
-echo %date% %time% ======================================================
-echo %date% %time% Regenerating Qt views
-echo %date% %time% ======================================================
-echo.
-sh scripts\rebuild_resources.sh
-if %errorlevel% neq 0 goto script_failure
+if "%regenerate_qt" == 1 (
```
Comment (mzieg) [inline scripts/bootstrap.bat:287]: I think this needs to be "1"

**Review by mzieg — CHANGES_REQUESTED** [review]

File: scripts/bootstrap.bat (hunk 1 above)
Comment (samiebee43) [reply scripts/bootstrap.bat:236]: it does

**Comment by samiebee43** [comment]
> Changes makes sense. My impression is that this will no longer have a "one-shot build installer" (equivalent to the old "installer" argument), but will rather require at least three invocations to match the old behavior (once with "rebuildall", then "pyinstaller", then "innosetup")?

Yes. I didn't include a one-shot, since that never worked on my end, but it can be trivially added

File: scripts/bootstrap.bat [hunk 3]
```diff
@@ -178,154 +248,142 @@ REM echo %date% %time% ======================================================
...
+)
 
-echo.
-echo %date% %time% ======================================================
-echo %date% %time% Regenerating Qt views
-echo %date% %time% ======================================================
-echo.
-sh scripts\rebuild_resources.sh
-if %errorlevel% neq 0 goto script_failure
+if "%regenerate_qt" == "1" (
```
Comment (mzieg) [inline scripts/bootstrap.bat:294]: This needs to be %regenerate_qt% (closing marker)

**Review by mzieg — CHANGES_REQUESTED** [review]

**Review by mzieg — APPROVED** [review]

**Comment by samiebee43** [comment]
This PR and branch build-stage are no longer needed since #120 is up to date with sbee-build

**Pull request closed without merging**